docs/devel/release.md

```diff
@@ -0,0 +1,64 @@
+# runtime-tools release guide
+
+## Release process
+
+This section shows how to perform a release of runtime-tools.
+The following example assumes we're going from version 0.5.0 (`v0.5.0`) to 0.6.0 (`v0.6.0`).
+
+Let's get started:
+
+- Start at the relevant milestone on GitHub (e.g. https://github.com/opencontainers/runtime-tools/milestones/v0.6.0): ensure all referenced issues are closed (or moved elsewhere, if they're not done). Close the milestone.
+- runtime-tools does not use a [roadmap file](https://github.com/opencontainers/runtime-tools/issues/465) but GitHub milestones. Update the [other milestones](https://github.com/opencontainers/runtime-tools/milestones), if necessary
+- Branch from the latest master, make sure your git status is clean
```

Comment:
I think we want backticks for `git status`.

Also, some of these list entries end in periods (e.g. “Close the milestone.”) and some don't (e.g. this one).  I think we should pick whether we want list entries to include trailing punctuation and then be consistent about it.  My personal preference is for trailing punctuation, but I'm fine with either approach as long as we're consistent.